CMakeLists.txt

```diff
@@ -5,6 +5,8 @@
 #
 include(common.cmake)
 
+message("HELLO FROM NRFXLIB")
```

Comment:
Is that change really intended?

Reply:
No, these should not be there :/ Updated it now!